core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/function/object/ObjectLengthExpression.kt

```diff
@@ -0,0 +1,12 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.function.`object`
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.function.FunctionExpression
+import ch.ergon.dope.validtype.NumberType
+import ch.ergon.dope.validtype.ObjectType
+
+class ObjectLengthExpression(
+    objectExpression: TypeExpression<ObjectType>,
+) : FunctionExpression<NumberType>("OBJECT_LENGTH", objectExpression)
+
+fun TypeExpression<ObjectType>.length() = ObjectLengthExpression(this)
```

Comment:
I think you should be consistent with arrays. Using arrays we don't use extension-functions and name it _arrayLength_. It's confusing for the user if we change the syntax always

Reply:
(Comment is for all the object-functions)